web/client/components/map/cesium/Map.jsx

```diff
@@ -190,6 +190,19 @@ class CesiumMap extends React.Component {
             };
             this.setView(position);
         }
+
+        if (prevProps && (!isEqual(this.props.mapOptions.showSkyAtmosphere, prevProps?.mapOptions?.showSkyAtmosphere))) {
```

Comment:
I'm not sure we need is equal for this options comparison, shouldn't be all boolean? same for these others new props